tools/api-tests/src/xpub-api/submission.ts

```diff
@@ -255,9 +256,21 @@ const fileUploadProgress = async (_ctx: ApiTestContext): Promise<Manuscript> =>
   throw new NotImplementedError();
 };
 
+// Submits a survey
+const submitSurvey = async (ctx: ApiTestContext, data: SurveySubmission): Promise<{ submitSurvey: boolean}> => {
+  const query = `
+  mutation SubmitSurvey($data: SurveySubmission!) {
+  submitSurvey(data: $data)
+}
+  `;
+
+  return await withAuthorization(ctx, query, {data} );
+}
+
 const resolvers = {
   Query: { manuscript, editors, getSubmission},
   Mutation: {
+    submitSurvey,
```

Comment:
Because we've packages the survey in `component-survey` instead of `component-submission` I think this needs moving to its own `survey.ts` file so the list of `Mutations` here is correct

Reply:
but the survey resolver is inside `component-submission` - we haven't packaged the survey in `component-survey`

Reply:
Resolved face-to-face